src/app/shared/model/distribution.model.ts

```diff
@@ -40,46 +42,62 @@ export class DistributionModel extends BaseModel {
   }
 
   related() {
+    let versionTemplate = Observable.of(null);
     let podcast = Observable.of(null);
-    let versionTemplate: Observable<AudioVersionTemplateModel> = Observable.of(null);
-
-    if (this.isNew) {
-      if (this.parent && this.parent.has('prx:account')) {
-        podcast = this.parent.follow('prx:account').map(account => {
-          let podmodel = new FeederPodcastModel(this.parent, this.doc);
-          if (account && account['name']) {
-            podmodel.set('authorName', account['name'], true);
-          }
-          return podmodel;
-        });
-      } else {
-        podcast = Observable.of(new FeederPodcastModel(this.parent, this.doc));
-      }
+
+    // set defaults from series for new podcasts
+    if (this.isNew && this.parent) {
+      podcast = this.parent.follow('prx:account').map(account => {
+        let podmodel = new FeederPodcastModel(this.parent, this.doc);
+        if (account && account['name']) {
+          podmodel.set('authorName', account['name'], true);
+        }
+        return podmodel;
+      });
+    }
+
+    // load existing podcasts
+    if (this.kind === 'podcast' && this.url) {
+      podcast = this.doc.followLink({href: this.url}).map(pdoc => {
+        return new FeederPodcastModel(this.parent, this.doc, pdoc);
+      });
     }
 
+    // load existing version templates
     if (this.doc && this.doc.has('prx:audio-version-template')) {
       versionTemplate = this.doc.follow('prx:audio-version-template').map(tdoc => {
         return new AudioVersionTemplateModel(this.parent, tdoc);
       });
     }
-    return {
-      podcast,
-      versionTemplate
-    };
+
+    return {podcast, versionTemplate};
   }
 
   decode() {
     this.id = this.doc['id'];
     this.kind = this.doc['kind'] || '';
+
+    // TODO: cms has non-auth'd urls
     this.url = this.doc['url'] || '';
+    if (this.url && !this.url.match('/authorization/')) {
```

Comment:
Feeder was returning 404s when the story was unpublished, so i switched to the `/authorized` url for the story.  I think there's a fix out so Feeder will actually redirect these requests.  But I'll check that later.